e2e/nomostest/nt.go

```diff
@@ -386,6 +390,7 @@ func DefaultRepoSha1Fn(nt *NT, nn types.NamespacedName) (string, error) {
 func (nt *NT) RenewClient() {
 	nt.T.Helper()
 	nt.KubeClient = newTestClient(nt)
+	nt.Watcher = testwatcher.NewWatcher(nt.Context, nt.Logger, nt.KubeClient, nt.Config, nt.Scheme, nt.DefaultWaitTimeout)
```

Comment:
nit: testwatcher.New should return a TestWatcher

Reply:
I guess testwatcher.NewWatcher returning a Watcher is ok... It's not watching the test, i guess.